### PR TITLE
Mfranz1/competency client/rouge one

### PR DESCRIPTION
**Pull request by mfranz1**
[Relates to story [22522](https://app.shortcut.com/clarkcan/story/22522/rouge-one-a-competencies-story)
# Description
Against General Zaleppa's orders, I have proceeded with infiltrating the Competency Star base in order to improve quality of life, provide the minimum needed functionality for beta, and also got the plans for the death star.

# Here's my hope
* Help pages are implemented
* Footer has been added
* Login/Registration forms updated
* Competency text field component implemented
* Logos (CC and partners)
* Minor styling changes
* LoggedIn/Out logic for navbar and help pages
* Implemented safe-html pipe

# WIP
* Copy for help pages

### CODE REVIEW

File: src/app/shared/components/footer/footer.component.scss [hunk 1]
```diff
@@ -0,0 +1,151 @@
+@import 'vars.scss';
+@import url('https://fonts.googleapis.com/css?family=Open+Sans');
+
+footer {
+  width: 100%;
+  color: whitesmoke;
+  background: #376ED6;
```
Comment (pzalep1) [inline src/app/shared/components/footer/footer.component.scss:7]: Is this a variable color?

File: src/app/shared/components/footer/footer.component.scss [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        transform: scale(0.95);
+      }
+  
+      &.twitter {
+        background: #1da1f2;
+      }
+  
+      &.linkedin {
+        background: #0077b5;
+      }
```
Comment (pzalep1) [inline src/app/shared/components/footer/footer.component.scss:104]: Are we sharing the SecurEd ones?

File: src/app/shared/pages/help-page/copy.ts [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+                        answer: '<p>The competency builder Actor form has entry fields <b>Actor</b> and <b>Description</b>. ' +
+                            `Select an <b>Actor</b> from the provided list that best fits the individual that would be attempting ` +
+                            `to achieve this competency. The <b>Description</b> field allows you to enter additional information that ` +
+                            `may be previlent to defining your actor (i.e. prerequiste classes, competition details, etc...).</p>`
+                    },
+                    {
+                        question: 'Building Behavior',
+                        answer: `<p>The competency builder Behavior form has entry fields <b>Workrole</b>, <b>Task</b>, ` +
+                            `and <b>Task Description</b>. You can search and select a <b>Workrole</b> which is a NICE work role*. ` +
+                            `<b>Task</b>'s can also be searched and added as a relating task of this competency. These are also ` +
```
Comment (pzalep1) [inline src/app/shared/pages/help-page/copy.ts:39]:
```suggestion
                            `<b>Task</b>'s can also be searched and added to the competency here. These are also ` +
```

File: src/app/shared/components/footer/footer.component.scss (hunk 2 above)
Comment (mfranz1) [reply src/app/shared/components/footer/footer.component.scss:104]: Let's discuss this for alpha. I deleted them for now. They have the info email and secured site

File: src/app/shared/components/preview-competency/preview-competency.component.html [hunk 4]
```diff
@@ -1,113 +1,112 @@
...
-       
+  <div class="header" *ngIf="!isAdmin else adminHeader" [ngClass]="{'builder-content': builderMode === true}"></div>
+  <div class="content">
+    <!-- TODO: Implement once user routes and submission dates are implemented -->
+    <!-- <div *ngIf="isAdmin" class="details bottom-bordered content-section">
+      <h2>Details</h2>
+      <p><b>Date Submitted:</b> Decembruary 32, 1932</p>
+      <p><b>Author Name: </b> Mick E. Maus</p>
+      <p><b>Author Email: </b> mickemaus@aol.com</p>
+    </div> -->
```
[the remark this answers is not in the document]
Comment (mfranz1) [reply src/app/shared/components/preview-competency/preview-competency.component.html:10]: Yee admin preview PR paulo's working on

File: src/app/shared/components/footer/footer.component.scss (hunk 1 above)
Comment (mfranz1) [reply src/app/shared/components/footer/footer.component.scss]: I updated it to one

File: src/app/core/builder.service.ts [hunk 5]
```diff
@@ -53,6 +53,10 @@ export class BuilderService {
     }
     // Updates current builder index
     public setBuilderIndex(value: number) {
+        // Disable Rubrics for degree
+        if (value === 6 ) {
```
Comment (pzalep1) [inline src/app/core/builder.service.ts:57]:
```suggestion
        if (value === 6) {
```

File: src/app/shared/components/footer/footer.component.html [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+          </a>
+        </div>
+        <div class="logo-wrapper">
+          <a href="https://www.secured.team/" target="_blank">
+            <img src="../../../../assets/secured.png" alt="SecurEd Inc. Logo" style="width: 115px;">
+          </a> 
+        </div>
+      </div>
+    </div>
+    <p>{{ copy.SUPPORTEDBY }}</p>
```
Comment (pzalep1) [inline src/app/shared/components/footer/footer.component.html:45]: Did the extra under get deleted from this?

**Review by Cwagne17 — APPROVED** [review]

**Review by pzalep1 — APPROVED** [review]

File: src/app/shared/components/primary-nav/primary-nav.component.ts [hunk 7]
```diff
@@ -1,4 +1,5 @@
 import { Component, EventEmitter, OnInit, Output } from '@angular/core';
+import { AuthService } from '../../../core/auth.service';
```
Comment (pmanal1) [inline src/app/shared/components/primary-nav/primary-nav.component.ts:2]: Delete if not being used

**Review by pmanal1 — APPROVED** [review]

**Pull request merged**